demos/twitter-race/README.md

```diff
@@ -0,0 +1,46 @@
+# Twitter Race
+
+This example shows reading from a live stream of tweets using the [twitter adapter](https://github.com/juttle/juttle-twitter-adapter).
+The user inputs 2 search terms, juttle parses the incoming tweets to find matches, and displays match count as live-updating tiles and a timechart,
+to visually compare popularity of the requested terms.
+
+We will need outrigger to render visualizations in the browser, and the twitter adapter to enable reading. The outrigger docker image provides the necessary software. We run it with specific options that map the necessary configuration and juttle programs into the container.
+
+## Setup
+
+### Twitter Adapter Configuration and Credentials
+
+You should create a `juttle-config.json` file in the current directory. This should contain a configuration section with credentials to access twitter via API:
+
+```
+{
+    "adapters": {
+        "juttle-twitter-adapter": {
+            "consumer_key": "...",
+            "consumer_secret": "...",
+            "access_token_key": "...",
+            "access_token_secret": "..."
+        }
+    }
+}
+```
+
+To obtain these credentials, set up a [Twitter App](https://apps.twitter.com/) using your twitter username, and create an OAuth token.
+
+### Starting outrigger
+
+Start outrigger via docker:
+
+```
+docker run --name outrigger -p 8080:8080 -v `pwd`/juttles:/opt/outrigger/juttles -v `pwd`/juttle-config.json:/opt/outrigger/.juttle-config.json juttle/outrigger:latest
+```
```

Comment:
Add a note that if you're already running a docker outrigger demo, you should change the command to `--name outrigger1 -p 8081:8080` or similar, i.e. change both the container name, and the local port.